backend/src/main/kotlin/hu/bme/sch/cmsch/component/token/TokenEntity.kt

```diff
@@ -102,6 +102,20 @@ data class TokenEntity(
     @property:ImportFormat(ignore = false)
     var displayDescription: String = "",
 
+    @field:JsonView(value = [ Edit::class, Preview::class, FullDetails::class ])
+    @Column(nullable = true)
```

Comment:
Add columnDefinition with default value to make it migrate itself into the db schema.

Reply:
Done